apps/survey/contracts/Survey.sol

```diff
@@ -109,7 +108,7 @@ contract Survey is AragonApp {
         survey.metadata = _metadata;
         survey.snapshotBlock = getBlockNumber() - 1; // avoid double voting in this very block
         survey.votingPower = token.totalSupplyAt(survey.snapshotBlock);
-        require(survey.votingPower > 0);
+        assert(survey.votingPower > 0);
```

Comment:
Why the change to `assert`? Should still be `require` as the token supply could be 0 when a new survey is created, no need to waste all the gas for something that is not an internal consistency.